File: single-language-analysis/dataset-files/python/test_smell_143.py
# Original URL: https://github.com/infstellar/genshin_impact_assistant/blob/9d3357bbe6c50b5aff410d5695da4bd52e0dc26e/source/ocr/test_ocr.py#L6-L10

def test_smell_143():
    from source.ocr.ocr import Ocr
    ocr = Ocr()
    logger.info("OCR RESULT: "+str(ocr.ocr_single_line(cv2.imread(fr'{ROOT_PATH}/assets/pytest/AreaBigmapSidebarCommissionName.jpg'))))
    assert 1

File: single-language-analysis/dataset-files/java/TestSmell143.java
import org.junit.Test;
import static org.junit.Assert.*;
import source.ocr.Ocr;

public class TestSmell143 {

    @Test
    public void testOcrAvailability() {
        Ocr ocr = new Ocr();
        System.out.println("OCR RESULT: " + ocr.ocrSingleLine(cv2.imread(ROOT_PATH + "/assets/pytest/AreaBigmapSidebarCommissionName.jpg")));
        assertEquals(1, 1);
    }
}